docs/configuration/index.md

```diff
@@ -2103,18 +2103,18 @@ In addition to the normal cache metrics, the caffeine cache implementation also
-| `druid.cache.memcachedPrefix` | Key prefix for all keys in Memcached.                                                                          | druid             |
-| `druid.cache.numConnections`  | Number of memcached connections to use.                                                                        | 1                 |
-| `druid.cache.protocol`        | Memcached communication protocol. Can be binary or text.                                                       | binary            |
-| `druid.cache.locator`         | Memcached locator. Can be consistent or array_mod.                                                             | consistent        |
-| `druid.cache.enableTls`       | Use TLS based ssl connection for Memcached client.  Boolean                                                    | false             |
-| `druid.cache.clientMode`      | ClientMode. Dynamic mode uses AutoDiscovery feature of AWS Elasticache Memcache. String. "static" or "dynamic" | static            |
+| Property                      | Description                                                                                            | Default           |
+|-------------------------------|--------------------------------------------------------------------------------------------------------|-------------------|
+| `druid.cache.expiration`      | Memcached [expiration time](https://code.google.com/p/memcached/wiki/NewCommands#Standard_Protocol).   | 2592000 (30 days) |
+| `druid.cache.timeout`         | Maximum time in milliseconds to wait for a response from Memcached.                                    | 500               |
+| `druid.cache.hosts`           | Comma separated list of Memcached hosts `<host:port>`.                                                 | none              |
+| `druid.cache.maxObjectSize`   | Maximum object size in bytes for a Memcached object.                                                   | 52428800 (50 MiB) |
+| `druid.cache.memcachedPrefix` | Key prefix for all keys in Memcached.                                                                  | druid             |
+| `druid.cache.numConnections`  | Number of memcached connections to use.                                                                | 1                 |
+| `druid.cache.protocol`        | Memcached communication protocol. Can be binary or text.                                               | binary            |
+| `druid.cache.locator`         | Memcached locator. Can be consistent or array_mod.                                                     | consistent        |
+| `druid.cache.enableTls`       | Use TLS based ssl connection for Memcached client.  Boolean                                            | false             |
+| `druid.cache.clientMode`      | Client Mode. Dynamic mode uses AutoDiscovery feature of AWS Memcached. String. "static" or "dynamic"   | static            |
```

Comment:
two things:
- can we explain that "static" means individual cluster nodes have to be configured?
- can we add a link to the AWS docs to mention how to configure the `druid.cache.hosts` when using dynamic mode?